auth-web/src/views/auth/staff/ReviewAccountView.vue

```diff
@@ -12,7 +12,7 @@
       <!-- Breadcrumbs / Back Navigation -->
       <nav class="crumbs py-6">
         <div>
-          <router-link to="/searchbusiness">
+          <router-link :to="accountUnderReview.statusCode === 'REJECTED' ? '/staff-dashboard/rejected' : '/staff-dashboard/review'">
```

Comment:
Can we use the const variable instead of the string 'REJECTED'?

Reply:
or a computed get value for the check

Reply:
yup.I noticed that as well.. will put it in my next PR
in the next PR ,i am changing all urls to constants.. so goes along with that